cmd/maya-apiserver/cstor-operator/cspc/start_test.go

```diff
@@ -24,7 +25,8 @@ func TestStart(t *testing.T) {
 	var err error
 	var errchannel = make(chan error)
 	go func() {
-		err = Start()
+		var mux = sync.RWMutex{}
```

Comment:
Please explain the need for mutex in PR message.

Reply:
Done